mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -371,6 +374,8 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 	METHOD m_hehkhdfc isFrozen ()Z
 	METHOD m_hhmmivsh tryTickInVoid ()V
 	METHOD m_hhwoykag calculateNextStepSoundDistance ()F
+	METHOD m_hisokqyg setOnFireForTicks (I)V
```

Comment:
```suggestion
	METHOD m_hisokqyg setOnFireFor (I)V
```
I'd prefer to just name this `setOnFireFor`, and rename `setOnFireFor` to `setOnFireForSeconds`. ticks are the more common unit in MC code, so it makes sense to me

Reply:
Done!